tools/cordova/index.js

```diff
@@ -14,7 +14,7 @@ export const CORDOVA_DEV_BUNDLE_VERSIONS = {
 };
 
 export const CORDOVA_PLATFORM_VERSIONS = {
-  'android': '6.3.0',
+  'android': '6.4.0',
```

Comment:
If I'm understanding https://github.com/apache/cordova-android/pull/417 correctly this version update will break crosswalk compatibility. It *seems* that `6.3.0` and `7.0` will support crosswalk again (unverified). 

It might be worth to mention this in the changelog if that's the case, if not sorry for the false alarm.

*Note: There are still a lot of projects which rely on crosswalk as not every targeted device has an updated android system webview. We use Meteor/Crosswalk to also release our project on smartglasses which have some sort of un-updateable webview in general*

Reply:
From what I could tell, `cordova-android@6.3.0` is totally broken (as in, you can't `meteor run android` at all) by the missing `android-versions` package.

Possible solutions:

1. Is there an earlier version we could pin, instead of 6.{3,4}.0?
2. Are we ready for 7.0.0?
3. Should we try manually merging https://github.com/apache/cordova-android/pull/417 and shipping that version of `cordova-android`?

Reply:
Went ahead with option #3, though it's not too late for #1 if that seems safer. #2 would probably need to wait for Meteor 1.6.2 at least.

Reply:
Imho you made the right decision. 

`7.0.0` has too many changes and will potentially delay `Meteor 1.6.1` a lot. Cherry picking was the best approach in this situation, and to be honest, I would have expected `cordova-android 6.4.1` because of this in the first place.

*On a personal note: Javascript is sometimes a crazy and scary world including big challenges for companies to keep up with all the changes. While `cordova-android` omitted that challenge in this specific case, it's a relief to see that Meteor has that attitude in mind. Thank you very much for your efforts on this @benjamn, I highly appreciate that.*